src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateActionPlanTest.kt

```diff
@@ -188,8 +188,10 @@ class CreateActionPlanTest : IntegrationTestBase() {
     assertThat(actionPlan)
       .isForPrisonNumber(prisonNumber)
       .hasNumberOfGoals(2)
-      // Because the goals are returned in date created order, the first goal in the response will be Goal 2, because it was the most recently created.
-      .goal(1) {
+      // Because the goals are returned in date created order, the first goal in the response SHOULD be Goal 2, because it was the most recently created.
```

Comment:
Not related to the main changes in this PR, but seeing as I had a failed build I took the opportunity to fix this flaky test which, as we discovered yesterday, is flaky because the order of the goals cannot be asserted reliably.